modules/api/src/main/TextLpvExpand.scala

```diff
@@ -59,7 +59,7 @@ final class TextLpvExpand(
     }
 
   private val gameRegex =
-    s"""/(?:embed/)?(?:game/)?(\\w{8})(?:(?:/(white|black))|\\w{4}|)(#\\d+)?\\b""".r
+    s"""[:.]\\w{1,5}/(?:embed/)?(?:game/)?(\\w{8})(?:(?:/(white|black))|\\w{4}|)(#\\d+)?\\b""".r
```

Comment:
This breaks forum embeds since Lichess links there don't include the domain. Reverted this part in https://github.com/lichess-org/lila/commit/4d53f7ba57cbeadae41c9e8494bc154fd1e7e94a but if you added it for a reason other than avoiding accidental embeds (or there's a situation where that frequently happens?) it might need some alternative changes.

Reply:
It's not necessary.  I added it because profile links and things like tournament/abc123XZ were matching and causing spurious game db lookups.

Reply:
I think maybe you are confusing scheme with domain here.

The regex change I made just insures the pre-parser matches .org or :9663 just prior to the game id.  What exactly did you see it break?

Reply:
Oh also, when are links of the form */embed/* and */game/* used?

Lastly, should someone tell the content team that they can link games again if it's not a legacy thing?  (I noticed they've been exclusively linking studies the past 5 months so I made a bad assumption it was an abandoned feature).

Reply:
Yeah, good point, I'll mention it.

And I guess actually, maybe just `(?:^|[:.]\\w{1,5})` works as an alternative? And I guess the `/game` is only there for `/embed/game/`, probably should just be nested within that.

Reply:
/embed would never be used for lpvs though, correct?

Reply:
Well, the regex detects user links in forums, ublogs, etc. So currently, if somebody posts an embed link, it will expand. Though I guess it's slightly debatable whether they should expand and it's probably not commonly used like that. But maybe some people do it because they think it's necessary or somehow get the link via the "embed on your website" thing or something (though that gives you iframe code).